docs/en/advanced-concepts/memory-collection.md

```diff
@@ -21,6 +21,21 @@ kill -SIGUSR2 &lt;nodePID&gt;
 Dump files are created in the same folder where the main Appium script has been executed.
 They have the `.heapsnapshot` extension, and can be loaded into the Chrome Inspector for further investigation.
 
+#### Examples
+```shell
+# Go to the directory where appium is installed via npm. 
+cd /home/k/.npm-global/lib/node_modules/appium/
```

Comment:
`cd "$(npm -g root)/lib/appium/"` or `cd "$(npm root)/lib/appium/"`depending on which npm root is needed (global or local)

Reply:
How about `cd "$(npm -g root)/appium/"` or `cd "$(npm root)/appium"`?

```shell
# In my Manjaro linux where npm prefix is set
[k@a2600 ~]$ npm --version
6.14.7
[k@a2600 ~]$ npm config list | grep prefix
prefix = "/home/k/.npm-global"
[k@a2600 ~]$ echo "$(npm -g root)"
/home/k/.npm-global/lib/node_modules
[k@a2600 ~]$ ls "$(npm -g root)"
appium  appium-base-driver  appium-doctor  face  mjpeg-consumer  node  node-gyp  opencv4nodejs


# In my Manjaro linux where npm prefix is not set
[k@x15 ~]$ npm --version
6.14.7
[k@x15 ~]$ echo "$(npm -g root)"
/usr/lib/node_modules
```

Reply:
^ what is the issue with the commands above? Everything looks correct

Reply:
I mean use `cd "$(npm -g root)/appium/"` instead of `cd "$(npm -g root)/lib/appium/"`, because `$(npm -g root)` already contains the path `/lib/node_modules`. And directory `appium` is directly under `/lib/node_modules`.

Reply:
ah, ok, this was actually a copypaste typo from my side